exchange/utils.go

```diff
@@ -64,6 +75,13 @@ func cleanOpenRTBRequests(ctx context.Context,
 		LMT:   lmtPolicy.ShouldEnforce(),
 	}
 
+	if gdpr == 1 {
+		cleanMetrics.gdprEnforced = true
```

Comment:
Cool. So we're logging when enforcement is in play, but not if the enforcement resulted in action. Privacy enforcement still could be avoided for some bidders depending on permission setup. The same will soon be true for CCPA. I don't have a good suggestion here, just wanted to call it out.